src/azure/apim/ApimService.ts

```diff
@@ -26,6 +30,19 @@ export class ApimService {
         this.serviceName = serviceName;
     }
 
+    public async getService(): Promise<IApimServiceContract> {
```

Comment:
why do you need this? we can get this from node.data. let me share an example where you use this